adviser/base/argo-workflows/kebechet-run-results.yaml

```
@@ -0,0 +1,67 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: kebechet-run-results
+spec:
+  templates:
+    - name: schedule-kebechet
+      inputs:
+        parameters:
+          - name: "WORKFLOW_NAME"
+          - name: "THOTH_DOCUMENT_ID"
+          - name: "THOTH_ADVISER_METADATA"
+          - name: "origin"
+      container:
+        image: kebechet-job
+        volumeMounts:
+          - name: ssh-config
+            mountPath: /home/user/.ssh
+            readOnly: true
+        env:
+          - name: PIPENV_NOSPIN
+            value: "1"
+          - name: PIPENV_COLORBLIND
+            value: "1"
+          - name: PIPENV_HIDE_EMOJIS
+            value: "1"
+          - name: KEBECHET_SUBCOMMAND
+            value: run-results
+          - name: KEBECHET_REPO_URL
+            value: "{{inputs.parameters.origin}}"
+          - name: KEBECHET_SERVICE_NAME
+            value: "github"  # TODO: temp
+          - name: KEBECHET_ANALYSIS_ID
+            value: "{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+          - name: GITHUB_KEBECHET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: github-oauth-token
+                name: kebechet-secret
+          - name: GITLAB_KEBECHET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: gitlab-oauth-token
+                name: kebechet-secret
+          - name: PAGURE_KEBECHET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: pagure-oauth-token
+                name: kebechet-secret
+          - name: SENTRY_DSN
+            valueFrom:
+              secretKeyRef:
+                key: sentry-dsn
+                name: thoth
+          - name: THOTH_DEPLOYMENT_NAME
+            valueFrom:
+              configMapKeyRef:
+                key: deploymentName
```

```suggestion
                key: deployment-name
```
was changing in https://github.com/thoth-station/thoth-application/commit/50b4983283c8ebf6705f62b355dc1ff974ad6945